azure-functions/acbs-function/src/functions/acbs-amend-facility.js

```diff
@@ -142,11 +142,20 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
 
     yield context.df.Task.all([facilityCovenantRecord]);
 
+    // 4. SOF: Facility Guarantee Record (FCR)
```

Comment:
Is FCR correct - not sure on the ACBS/business terminologies here, but i'd expect the acronym to be `FGR` to match the title?

Reply:
reat spot